doc/engsys_resource_management.md

```diff
@@ -0,0 +1,94 @@
+## Resource management guidelines
+
+This document contains guidelines for creating and managing Azure resources in the Azure SDK Engineering System
+subscriptions. EngSys has automation that will delete resources based on the criteria outlined below. This is to
+improve our overall security and keep us in compliance with company policies.
+
+  * [Managing Dev Resources in the Playground Subscription](#managing-dev-resources-in-the-playground-subscription)
+     * [Resource Groups](#resource-groups)
+     * [Role Assignments](#role-assignments)
+  * [Managing CI resources in the test subscription](#managing-ci-resources-in-the-test-subscription)
+     * [Resource Groups](#resource-groups-1)
+     * [Role Assignments](#role-assignments-1)
+
+
+## Managing Dev Resources in the Playground Subscription
+
+This section applies to resource groups located in the `Azure SDK Developer Playground` subscription.
+
+Currently the automation will inspect resource groups and role assignments only.
+
+### Resource Groups
+
+The automation will delete all resource groups in the playground subscription DAILY, unless they meet one of the following criteria:
+
+- The resource group starts with a valid Microsoft alias within `microsoft.com` or `ntdev.microsoft.com`, followed by an
+  optional hyphen and extra characters.
+    - Valid group name examples: `myalias`, `myalias-test-foobar` for `myalias@microsoft.com`
+- The resource group contains a tag with the name `owners` where the value is a csv formatted string that contains at
+  least one valid Microsoft alias within `microsoft.com` or `ntdev.microsoft.com`. This convention should only be used
+  when it is not possible to name the resource group with an alias (for example, inner groups auto-created by a resource
+  provider like AKS).
+    - Valid owner tag examples: `owners: myalias`, `owners: myalias,anotheralias,lastalias`
+- The resource group contains a tag with the name `DeleteAfter` and an ISO8601 formatted date value, where the date is
```

Comment:
```suggestion
- The resource group contains a tag with the name `DeleteAfter` and an [ISO8601]( https://www.iso.org/iso-8601-date-and-time-format.html) formatted date value, where the date is
```

It may be helpful to include the link for those not familiar.   Not a huge fan of that landing page, but it is the official ISO source...

Reply:
Hmm, I'm torn here (and trying to find a possible better link). I worry that people would click this link trying to figure out the format without reading below to see the examples. Additionally, we also expect UTC (`Z` for zulu) otherwise you might have unintended results. Perhaps I should workshop this a little bit more to be extra clear. It'd be nice if there was some page where you could put in the date format and it would show you how to create it in a bunch of languages.

Reply:
Maybe just add an inline example of what the format looks like?   That would probably be enough to stop someone from worrying that they don't know what it means.